ProjectManager/ProjectManager/Source/Common/PlanManager.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ToDoManager.swift
+//  ProjectManager
+//
+//  Created by som on 2023/01/11.
+//
+
+import Foundation
+
+final class PlanManager: PlanManageable {
+    func create(title: String, description: String, deadline: Date) -> Plan? {
+        var plan: Plan?
+
+        plan = Plan(status: .todo,
+                    title: title,
+                    description: description,
+                    deadline: deadline,
+                    id: UUID())
+
+        return plan
```

Comment:
```suggestion
return .init(status: .todo, title: title, description: description, deadline: deadline, id: .init())
```

옵셔널, variable일 필요가 없어보여요.

Reply:
[2481053](https://github.com/yagom-academy/ios-project-manager/pull/247/commits/24810538d67ec721042aee5669421105f0d1e400)

넵 수정했습니다!